src/main/resources/db/migration/h2/V1__create_table_node.sql

```diff
@@ -0,0 +1,34 @@
+/*
+ *     Nuts discovery service for Corda network creation
+ *     Copyright (C) 2020 Nuts community
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+CREATE TABLE node
+(
+    id   INT AUTO_INCREMENT PRIMARY KEY,
+    hash VARCHAR(64)  NOT NULL UNIQUE,
```

Comment:
You're querying on hash and name, but they're not indexed, do we need them?

Reply:
Unique, so yes it has an index